File: PHP/PrimeNumber.php
<?php

$time_start = microtime(true); // start the timer

// Code to print the first 10000 prime numbers

set_time_limit(500); // set the time limit in order for the script to run properly

$ct=0;  
$n=0;  
$i=1;  
$j=1;

while($n<10000)  
{  
    $j=1;  
    $ct=0;  
    while($j<=$i)  
    {  
        if($i%$j==0)  
        $ct++;  
        $j++;   
        }  
        if($ct == 2)  
        {  
        echo $i;  
        echo "<br>";  
        $n++;  
    }  
    $i++;  
}

// Measurement

$memory_size = memory_get_usage(); // get the current memory usage

$memory_unit = array('Bytes','KB','MB','GB','TB','PB');

echo 'Used Memory : '.round($memory_size/pow(1024,($x=floor(log($memory_size,1024)))),2).' '.$memory_unit[$x]."<br>";

function win_sys_current_cpu_usage() { // get the current cpu usage
    $cmd = 'typeperf  -sc 1  "\Processor(_Total)\% Processor Time"';
    exec($cmd, $lines, $retval);
    if($retval == 0) {
      $values = str_getcsv($lines[2]);
      return floatval($values[1]);
    } else {
      return false;
    }
  }

echo 'Used CPU : ',win_sys_current_cpu_usage(),' %<br>';

$time_end = microtime(true); // end the timer
$execution_time = ($time_end - $time_start);
echo '<b>Total Execution Time:</b> '.$execution_time.' Seconds';

File: Python/PrimeNumber.py
import psutil
import time
import os

def my_func():

    # start the timer

    start_time = time.time()

    i=1
    x = 10000
    counter = 0

    # while loop to print the first 10000 prime numbers

    while True:
        c=0;
        for j in range (1, (i+1), 1):
            a = i%j
            if (a==0):
                c = c+1
        if (c==2):
            print (i)
            counter = counter + 1
            if counter >= x:
                break
        i=i+1

     # get and print the runtime

    print("--- %s seconds ---" % (time.time() - start_time))
    
    # get and print the current cpu usage

    print("CPU : %s" % psutil.cpu_percent())
    
    # get the current memory usage

    process = psutil.Process(os.getpid())
    
    # print the current memory usage

    print("Memory : %s bytes" % process.memory_info().rss)

if __name__ == '__main__':
    my_func()